<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>700</width><height>500</height></rect>
  </property>
  <property name="windowTitle"><string>Approve Users</string></property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">

    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text"><string>Pending User Approvals</string></property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
      <property name="font">
       <font><pointsize>18</pointsize><bold>true</bold></font>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QTableWidget" name="tableUsers">
      <column>
       <property name="text"><string>User ID</string></property>
      </column>
      <column>
       <property name="text"><string>Name</string></property>
      </column>
      <column>
       <property name="text"><string>Email</string></property>
      </column>
      <column>
       <property name="text"><string>Account Type</string></property>
      </column>
     </widget>
    </item>

    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnApprove">
        <property name="text"><string>Approve</string></property>
        <property name="minimumHeight"><number>40</number></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisapprove">
        <property name="text"><string>Disapprove</string></property>
        <property name="minimumHeight"><number>40</number></property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text"><string>Back</string></property>
      <property name="minimumHeight"><number>40</number></property>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
